docs/overview.md

```diff
@@ -0,0 +1,44 @@
+We believe KILT will be an essential building block of the Web 3.0. 
+
+In particular KILT provides:
+
+* A universal blockchain protocol for individuals, organisations, objects, and artificial intelligences to claim arbitrary attributes about themselves and get them attested by trusted entities.
+* Mechanisms for putting claim holders in control of their data by storing the information on their storage and by giving them the choice which information they want to disclose to whom.
+* A Trust Market for the Attesters of such claims, which allows trusted entities to attach prices to their valuable attestation work and get paid.
+
+Our main goal with KILT is to generate a level playing field for companies to explore new business models related to trust relationships and data sovereignty.
+With KILT we enable businesses and governments to rely on a common standard which is owned by everyone participating and not by a single company.
+
+## The problem
+
+For hundreds of years, identity and trust between entities has been organized in a decentralised way. 
+For example official, personal documents like passports were issued by trusted entities and handed out to the holders. 
+Those holders were then under full control of their credentials and could use them where ever they wanted. 
+This was a very simple and secure way to get information about a person and the person itself always had full control over it.
+
+This changed drastically when web2.0 evolved with all their centralised solutions like google, meta, twitter etc. 
+Due to the business model of those organisations our personal data, our identity, is now stored and under the control of the centralised service providers. 
+It's even worse because those companies do not only store the data out of necessity but also for the purpose of the business model.
+So now, everytime we log into one of this services we generate new data points about our identity and those are then aggregated and sold mostly for advertising purpuses.
+
+We want to change that and give the control over their data back to the users.
+
+## The solution
+
+We provide a protocol for managing your identity on the blockchain and to attach digital credentials to it which are attested by trusted entities. 
+Those credentials are not stored on the chain, but stay with the user, so they can be used for any purpose and always stay under their control. 
+This is somehow very similar to the approach used for centuries before our data was being monetarized by big corporations.
+
+The core ideas are:
+
+* Managing your identity on the blockchain in form of a [DID](https://w3c-ccg.github.io/did-spec/#id)
+* Claiming digital credentials and let them be attested 
+* Present and verify digital credentials
+* Don't store any personal data on the chain
+* Attestations are revocable by the attester
```

Comment:
I would like to normalize the kind/tense of the verbs we are using throughout the bullet points. One suggestion:
```suggestion
* Managing your identity on the blockchain in form of a [DID](https://w3c-ccg.github.io/did-spec/#id)
* Claiming digital credentials
* Facilitating revocable attestations
* Presenting and verifying digital credentials
* Not storing any personal data on the chain.
```